src/Symfony/Component/BrowserKit/Tests/AbstractBrowserTest.php

```diff
@@ -105,9 +105,9 @@ public function testGetRequestNull()
     public function testXmlHttpRequest()
     {
         $client = $this->getBrowser();
-        $client->xmlHttpRequest('GET', 'http://example.com/', [], [], [], null, true);
+        $client->xmlHttpRequest('GET', 'http://example.com/', [], [], [], '', true);
         $this->assertEquals($client->getRequest()->getServer()['HTTP_X_REQUESTED_WITH'], 'XMLHttpRequest');
-        $this->assertFalse($client->getServerParameter('HTTP_X_REQUESTED_WITH', false));
```

Comment:
I'm really unsure this is a safe change. It'd be more confident if we document the default as mixed.
Same about xmlHttpRequest: any changes to tests are red flags that require investigation.